app/Helpers/render/comment.php

```diff
@@ -79,22 +80,22 @@ function RenderCommentsComponent(
             $lastID = $comment['ID'];
         }
 
-        $canDeleteComment = $articleTypeID == ArticleType::User && $userID === $articleID || $permissions >= Permissions::Moderator;
+        $canDeleteComment = $articleTypeID == ArticleType::User && $user->id === $articleID || $permissions >= Permissions::Moderator;
 
         RenderArticleComment(
             $articleID,
             $comment['User'],
             $comment['CommentPayload'],
             // TODO no unix timestamp here
             (int) $comment['Submitted'],
-            $user,
+            $username,
             $articleTypeID,
             (int) $comment['ID'],
             $canDeleteComment
         );
     }
 
-    if (isset($user)) {
+    if (isset($user) && !$user->isMuted) {
```

Comment:
I wonder if a muted user should be informed so they don't question why they can't add comments?

Reply:
Updated in latest:

![Screenshot 2024-03-25 at 4 58 41 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/2685eba7-05dd-48f1-a321-97c438400fca)